src/utils/createSimpleRequirePropertyRule.ts

```diff
@@ -11,14 +25,40 @@ import { isJSONStringLiteral } from "./predicates.js";
  */
 export const createSimpleRequirePropertyRule = (
 	propertyName: string,
-	isRecommended = false,
+	{
+		ignorePrivateDefault = false,
+		isRecommended,
+	}: CreateRequirePropertyRuleOptions = {},
 ) => {
-	return createRule({
+	return createRule<Options>({
 		create(context) {
+			const enforceForPrivate =
+				context.settings.packageJson?.enforceForPrivate ??
+				"recommended";
```

Comment:
I'm not sure I understand why `enforceForPrivate` needs to have a `"recommended"` state.  If we have `undefined` (i.e. user didn't specify the setting), `true` (they want it enabled on everything), and `false` (they don't want it enabled), why do we need another `string` state?

Reply:
I agree. Initially I followed Josh's suggestion, but I think this is actually superfluous.